### PR TITLE
[tm only] TGU staging branch

### PR DESCRIPTION
**Pull request by Sadboysuss**
## About The Pull Request
I suspect the original pr can't be merged because there is over 10000 commits, this is all of them squash merged into one, let's see if this works.

### CODE REVIEW

File: fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm [hunk 1]
```diff
@@ -49,6 +49,8 @@
 	return TRUE
 
 /datum/action/cooldown/bloodsucker/feed/DeactivatePower()
+	if(!active)
+		return
```
Comment (JohnFulpWillard) [inline fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm:53]: This should never be the case, this is hiding an issue. Remove please.

**Comment by QuiteLiterallyAnything** [comment]
A couple things I noticed while playing on the server when this was test-merged:

1. Heliostation's public mining shuttle seems to have been replaced with regular one (or is otherwise not loading properly).
2. Round end sound effects/music now seem to include Fulpstation's round start music.

**Comment by FernandoJ8** [comment]
> A couple things I noticed while playing on the server when this was test-merged:
> 
>     1. Heliostation's public mining shuttle seems to have been replaced with a regular one (or it's otherwise not loading properly).
> 
>     2. Round end sound effects/music now seem to include Fulpstation's round start music.

On 1: It was previously the mining shuttle used by Northstar, which has been removed along with the map itself. Does it have much importance? We could always re-add it as helio's own mining shuttle, but if there isn't a good reason for it, I'd rather use the standard variant.

**Comment by FernandoJ8** [comment]
> A couple things I noticed while playing on the server when this was test-merged:
> 
>     1. Heliostation's public mining shuttle seems to have been replaced with a regular one (or it's otherwise not loading properly).
> 
>     2. Round end sound effects/music now seem to include Fulpstation's round start music.

On 2: That's config as far as I can tell, so admin issue.

File: fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm (hunk 1 above)
Comment (Sadboysuss) [reply fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm]: done

**Pull request closed without merging**